source/Calamari.AzureWebApp/Program.cs

```diff
@@ -17,6 +19,7 @@ protected override void ConfigureContainer(ContainerBuilder builder, CommonOptio
         {
             base.ConfigureContainer(builder, options);
             builder.RegisterType<ResourceManagerPublishProfileProvider>().SingleInstance();
+            builder.RegisterType<AzureContextScriptWrapper>().As<IScriptWrapper>().SingleInstance();;
```

Comment:
I am registering this wrapper explicitly? But default calamari only registers its own assembly + Calamari.Common.